src/main/resources/db/changelog/changelog-1.1.0.xml

```diff
@@ -0,0 +1,146 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-latest.xsd">
+
+    <property name="uuid_type" value="varbinary(16)" dbms="hana"/>
+    <property name="uuid_type" value="uuid" dbms="!hana"/>
+
+    <changeSet id="changeset-1.1.0_001" author="polinatonko">
+
+        <dropForeignKeyConstraint baseTableName="course_student" constraintName="fk_course_student_st"/>
+
+        <addColumn tableName="student">
+            <column name="new_id" type="${uuid_type}"/>
+        </addColumn>
+
+        <update tableName="student">
+            <column name="new_id" valueComputed="id"/>
+        </update>
+
+        <addNotNullConstraint tableName="student" columnName="new_id"/>
+
+        <dropPrimaryKey tableName="student" dropIndex="true"/>
+
+        <dropColumn tableName="student" columnName="id"/>
+
+        <addPrimaryKey tableName="student" columnNames="new_id"/>
+
+        <renameColumn tableName="student" oldColumnName="new_id" newColumnName="id"/>
+
+    </changeSet>
```

Comment:
empty lines could be deleted, as they can they are logically related to each other